clusterloader2/pkg/measurement/common/network/network_performance_measurement.go

```diff
@@ -396,3 +402,21 @@ func (npm *networkPerformanceMeasurement) getMetricData(data *testResultSummary,
 	data.DataItems = append(data.DataItems, metricDataItem)
 	klog.V(3).Infof("TestResultSummary: %v", data)
 }
+
+func (npm *networkPerformanceMeasurement) waitForMetricsFromPods() {
+	bufferTime := time.Duration(math.Max(10, float64(npm.numberOfClients)*podFactor)) * time.Second
+	timeout := initialDelayForTestExecution + npm.testDuration + bufferTime
+	interval := time.Duration(2) * time.Second
+	if err := wait.Poll(interval, timeout, npm.checkResponseReceivedFromPods); err != nil {
+		klog.Errorf("Failed to receive response from all pods: %v", err)
```

Comment:
Add a TODO to log which pods didn't receive success.

Also - log here how many pods didn't return results.

Reply:
modified